log/internal_logger.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"net/http"
+)
+
+var analyticsServerURL = "https://bitrise-step-analytics.herokuapp.com"
+
+// Message represents a line in a log
+type Message struct{
+	LogLevel string `json:"log_level"`
+	Message string `json:"message"`
+	Data map[string]interface{} `json:"data"`
+}
+
+func init() {
```

Comment:
other places we use Set functions to modify default values/configuration: https://github.com/bitrise-io/go-utils/blob/master/log/log.go#L13
I would go this way as `ANALYTICS_SERVER_URL` may a too common name, probably used by other tools, other than that it would be much clear what changed the analytics server URL.